### ISSUE TITLE
HW: Experiment with whether the MCU activity (computation, wifi) affects the temperature sensor

### ISSUE THREAD

**Issue by lars-t-hansen** [knowit/snappysense#43]
A simple experiment separates these by some distance to see if this affects the readings.

**Comment by lars-t-hansen** [comment]
The gnd is connected to the bottom copper layer on the circuit board and this plane acts as a heat sink, and this is connected to the top copper layer, and heat from that may affect the sensors.

**Comment by lars-t-hansen** [comment]
There's a fair amount of evidence that WiFi is a problem with the Arduino firmware; see investigation attached to #3.

There's also some evidence that the board itself is a problem even without WiFi.  With the IDF firmware and no WiFi, if the unit is placed on its side so that the MCU is oriented next to the board with its short end up to create a chimney effect around the MCU, the temperature readings seem to drop quite a bit.  But more experimentation is needed.

**Comment by lars-t-hansen** [comment]
I also did an experiment where I moved the CPU and display off-board, leaving only the DHT, AIR, and MIC sensors on-board.  With the IDF firmware (no WiFi), this setup produced temperatures 0.6 C lower than the regular v1.1 prototype unit turned on its side, 21.7C vs 22.3C.  The temperature in the room was about 20C, though, so even with these changes the readings seem high, but without measuring many more sensors standalone - easy to do, but time consuming - it's hard to know if this is a systematic error in all the sensors.

Anyway, I think that we're seeing a real problem here with activity on the board causing the temperature sensor to inflate its readings.  The main fix for this would be to move the sensor off-board.  If we're going to have an enclosure (#44) we may need to move things around in any case.
